portafly/src/components/util/AlertsContext.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react'
+import {
+  AlertGroup,
+  AlertActionCloseButton,
+  Alert,
+  AlertVariant
+} from '@patternfly/react-core'
+
+interface IAlert {
+  key: string
+  variant: keyof typeof AlertVariant
+  title: string
+}
+
+const AlertsContext = React.createContext<({ addAlert?: (alert: IAlert) => void })>({})
```

Comment:
`addAlert` is optional, that's causing the error in
https://github.com/3scale/porta/pull/1837/files#diff-7be801d10bf78bb72b655d866926c909R75

Reply:
I see.. I'll fix it